elytron/src/test/java/org/wildfly/extension/elytron/KeyStoresTestCase.java

```diff
@@ -430,6 +430,67 @@ public void testKeystoreCli() throws Exception {
         assertSuccess(services.executeOperation(operation));
     }
 
+    @Test
+    public void testKeystoreReadVerbose() throws Exception {
+        Path resources = Paths.get(KeyStoresTestCase.class.getResource(".").toURI());
+        Files.copy(resources.resolve("firefly.keystore"), resources.resolve("firefly-copy.keystore"), java.nio.file.StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Looks like `removeTestFiles` should be updated to delete `firefly-copy.keystore`.